Sources/SKTestSupport/SKTibsTestWorkspace.swift

```diff
@@ -40,8 +41,10 @@ public final class SKTibsTestWorkspace {
     tmpDir: URL,
     removeTmpDir: Bool,
     toolchain: Toolchain,
-    clientCapabilities: ClientCapabilities) throws
+    clientCapabilities: ClientCapabilities,
+    group: EventLoopGroup) throws
   {
+    self.testServer = TestSourceKitServer(group: group, connectionKind: .local)
     self.tibsWorkspace = try TibsTestWorkspace(
       immutableProjectDir: immutableProjectDir,
       persistentBuildDir: persistentBuildDir,
```

Comment:
@benlangmuir the reason for the test failure was that I removed line 52 (`removeTmpDir: removeTmpDir`) because locally I had an older version of the index store DB which didn't have this parameter. By accident, I recorded this change in my PR. The reason the test also failed on master is that I had to make this change to get it to compile. Now, I updated all swift package versions and it works everywhere...